app/pages/neworg/error/contextCreator.js

```diff
@@ -0,0 +1,11 @@
+import manifest from './manifest.json';
+import { baseUrl } from '../../../config';
+
+const getErrorMessage = ({ errorsString, odsCode }) => (errorsString.split('+').includes('OrganisationAlreadyExists')
+  ? `The organisation with ODS code ${odsCode} already exists.` : 'Something went wrong.');
+
+export const getContext = params => ({
+  ...manifest,
+  description: getErrorMessage(params),
+  backLinkHref: `${baseUrl}/organisations/find`,
```

Comment:
just on this. Should we passing in the odsCode with what they entered when they go back? or is Josh happy for it to go back and it to be empty

Reply:
yep I'll put it in

Reply:
actually it doesn't make sense to put it in as you would end up in the same place - on the error screen. I have added the ods code to the error screen so it can be viewed here

Reply:
josh says it is fine

Reply:
OK - I see what you mean.